electron-builder.ts

```diff
@@ -118,9 +116,6 @@ const config: Writable<Configuration> = {
         category: "Network;InstantMessaging;Chat",
         maintainer: "support@element.io",
         icon: "build/icons",
-        desktop: {
-            MimeType: "x-scheme-handler/element",
-        },
```

Comment:
This can be removed as electron-builder now correctly specifies it from the `protocol` definition lower down